python/tvm/generic.py

```diff
@@ -0,0 +1,24 @@
+"""Generic opertor interface for overloading.
+We follow the numpy naming convention for this interface
+(e.g., tvm.generic.multitply ~ numpy.multiply).
+"""
+# pylint: disable=unused-argument
+
+#Operator precedence used when overloading.
+__op_priority__ = 0
+
+def add(lhs, rhs):
+    """generic add interface"""
```

Comment:
add detailed dostring about parameters and return value. 

after this get replaced, inherit the dostring by assigning the ```__doc__``` field